lib/network.rb

```diff
@@ -0,0 +1,34 @@
+class Network
+  attr_reader :name, :shows
+
+  def initialize(name, shows = [])
+    @name = name
+    @shows = shows
+  end
+
+  def add_show(show)
+    @shows << show
+  end
+
+  def highest_paid_actor
+    highest_salary_name = ""
+    @shows.each do |show|
+      highest_salary_name = show.characters.max_by do |character|
+          character.salary
+      end
+    end
+    highest_salary_name.actor
```

Comment:
good job here, we'll cover an even shorter way to do this in the walkthrough tomorrow